translations/fr.json

```diff
@@ -1,339 +1,330 @@
   "tribes.elyrion": "∑∫ỹriȱŋ",
-  "tribes.elyrion.info": "Les mystérieux {0} défendent leur forêt avec une puissante magie et la férocité du feu des dragons ! \n\nLeur nature est sacrée, il ne chasse donc pas d'animaux et ne coupe pas d'arbres. Au lieu de cela, ils commencent la partie avec une technologie unique qui leur permet d'enchanter les animaux ordinaires et les transformer en monstres puissants.",
+  "tribes.elyrion.info": "Les mystérieux {0} défendent leurs maisons boisées avec la magie colorée et la férocité des dragons cracheurs de feu ! \n \nIls considèrent la nature comme un esprit saint et ne peuvent pas chasser les animaux ni abattre les arbres, à la place ils commencent le jeu avec la technologie unique d'enchantement qui peut transformer des animaux ordinaires en monstres puissants.",
   "tribes.yadakk": "Yădakk",
-  "tribes.yadakk.info": "Les {0} commencent comme tribu nomade dans les merveilleuses plaines de Khalee. Désormais, ils sont les commerçants du monde, reliant leur empire à des routes commerciales impressionnantes. \n\nLes {0} commencent la partie avec la technologie '{1}'.",
+  "tribes.yadakk.info": "Les {0} ont commencé comme une tribu nomade dans les belles plaines de Khalee. Maintenant, ils sont les commerçants de la place, reliant leur empire à des routes commerciales impressionnantes. \n \n {0} démarre le jeu avec la technologie '{1}'. ",
   "tribes.polaris": "Polaris",
-  "tribes.polaris.info": "Les {0} sont restés piégés dans les confins de la toundra glaciale depuis des siècles, mais ils ont été bénis par le mystérieux Gaami qui leur donna le pouvoir d'étendre leur terrain glacial artificiel plus loin que le temps ne le permet. \n \nAvec la puissance des traîneaux et des Mammouths, les mystiques {0} sont déterminés à enfouir le monde dans la glace et à transformer la terre en un paradis glacial. \n \nLes {0} démarrent la partie avec la possibilité de geler le terrain environnant en utilisant des Moonies.",
-  "building.capital.owner": "{0} est la capitale de {1} .",
-  "building.capital.owner.former": "{0} est l'ancienne capitale de l'empire {1} , actuellement occupée par les forces {2} .",
-  "building.city.owner": "{0} est une ville de l'empire {1} .",
+  "tribes.polaris.info": "Les {0} sont restés coincés dans les confins de la toundra glaciale pendant des siècles, mais ont été bénis par l'inconnu Gaami avec le pouvoir d'étendre leur terrain glacial artificiel plus loin que le temps ne le permet. \n \nAvec la puissance des traîneaux et des Mamoos, les {0} mystiques sont déterminés à enfouir ces terre dans la glace et à transformer la terre en un paradis glacial. \n \n {0} commence le jeu avec la possibilité de geler le terrain environnant en utilisant Moonies. ",
+  "building.capital.owner": "{0} est le {1} capital",
```

Comment:
In french we say "Paris est la capitale de la France"... Not "Paris est le France capital"